modules/appeals_api/app/models/appeals_api/notice_of_disagreement.rb

```diff
@@ -255,6 +256,20 @@ def validate_hearing_type_selection
       end
     end
 
+    # v2 specific validation
+    def validate_extension_request
```

Comment:
I'm torn between whether we should do this or just document that we'll ignore the extensionReason if extensionRequest is false. I'll have to look over some of our other validations to see if anything sways me. Not a blocker for now, but wanted to mention where my head's at so it doesn't come as a surprise if we remove it later.

Reply:
I asked in our bp private channel and kelly responded that we should error. I'll bring it up during stand up next week.